.github/actions/yarn-install/action.yaml

```
@@ -0,0 +1,28 @@
+name: "My Shared Steps"
```

Er ikke dette et mer beskrivende navn? 🤔 
```suggestion
name: "Yarn Install with cache"
```

---

Jo, absolutt! :D Ble litt ivrig i copy pastingen der, i Altinn Studio repoet står det My Shared Steps. 😇 Fikser! :)